### PR TITLE
AAF Clan skins / NRL Clothing [Ready for Review]

### PR DESCRIPTION
**Pull request by Tonnie15**

### CODE REVIEW

**Comment by Joshua-Brown** [comment]
this is a comment

**Comment by Tonnie15** [comment]
Can I get someone to take a look over the config.cpp and make sure everything is sweet please

File: UB_Fixes/config.cpp [hunk 1]
```diff
@@ -24,7 +24,11 @@ class CfgPatches
 	    "C_man_polo_2_Socceroos",
 	    "C_man_polo_2_Dkfc",
 	    "C_man_polo_2_F_Subby",
-	    "B_AOR2_F_AL"
+	    "B_AOR2_F_AL",
+		"UB_NRL_broncos",
+		"UB_NRL_rabbitohs",
+		"UB_NRL_roosters"
+
```
Comment (fresqo) [inline UB_Fixes/config.cpp]: remove blank line please.

File: UB_Fixes/config.cpp [hunk 2]
```diff
@@ -24,7 +24,11 @@ class CfgPatches
 	    "C_man_polo_2_Socceroos",
 	    "C_man_polo_2_Dkfc",
 	    "C_man_polo_2_F_Subby",
-	    "B_AOR2_F_AL"
+	    "B_AOR2_F_AL",
+		"UB_NRL_broncos",
```
Comment (fresqo) [inline UB_Fixes/config.cpp]: adjust spacing alignment please

File: UB_Fixes/config.cpp [hunk 3]
```diff
@@ -38,6 +42,9 @@ class CfgPatches
 			"RH_fn57_fire",
 			"NovusCopHorn",
 			"srifle_EBR_F_CITY",
+			"UB_NRL_broncos",
+			"UB_NRL_rabbitohs",
+			"UB_NRL_roosters"
```
Comment (fresqo) [inline UB_Fixes/config.cpp]: missing comma

File: UB_Fixes/config.cpp (hunk 2 above)
Comment (Joshua-Brown) [inline UB_Fixes/config.cpp]: Don't have them indented.

File: UB_Fixes/config.cpp (hunk 3 above)
Comment (Joshua-Brown) [inline UB_Fixes/config.cpp]: missing "," on line 47

**Comment by Joshua-Brown** [comment]
thats all i can see from a quick look, ill get @fresqo to have a look also

**Comment by Tonnie15** [comment]
Please review again when ready

**Pull request merged**